Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -381,6 +381,29 @@ def shuffle_split_changed(self):
     def _param_changed(self):
         self._invalidate()
 
+    def push_results(self, learners, results, class_var):
```

Comment:
I would put this method below _update_results and pick another name. A docstring might be a good idea as well, as it is not that obvious what the method does.